website/content/en/docs/upgrading/v1-migration.md

```diff
@@ -229,9 +229,11 @@ for [EKS Pod Identity ABAC policies](https://docs.aws.amazon.com/eks/latest/user
    ```
 
 * **amiSelectorTerms and amiFamily**: For v1, `amiFamily` is no longer required if you instead specify an `alias` in `amiSelectorTerms` in your `EC2NodeClass`. You need to update your `amiSelectorTerms` and `amiFamily` if you are using:
-   * A Custom amiFamily. You must ensure that the node you add the `karpenter.sh/unregistered:NoExecute` taint in your UserData.
+   * A Custom amiFamily. 
    * An Ubuntu AMI, as described earlier.
 
+* **userData**: If you configure [`userData` block](https://karpenter.sh/preview/concepts/nodeclasses/#specuserdata), regardless of the amiFamily used, you must ensure to add the `karpenter.sh/unregistered:NoExecute` taint in your UserData.
```

Comment:
Can we make this a local reference (there's examples of this throughout our docs)